pegasus/test/test_request.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../../lib/cdo/rack/request'
```

Comment:
Keeping this in pegasus/test for now where most other tests for lib/cdo currently live because an implicit association between lib/ and shared/ seems more confusing. If we want to unify shared/ and lib/ then we should decide that, otherwise if we want to improve the organization of the tests then tests like this could all go in lib/test.